packages/mobile/src/components/core/Screen/ScreenHeader.tsx

```diff
@@ -0,0 +1,108 @@
+import type { ComponentType, ReactChild } from 'react'
+
+import { FeatureFlags } from '@audius/common'
+import type { TextStyle, ViewStyle } from 'react-native'
+import { View } from 'react-native'
+import type { SvgProps } from 'react-native-svg'
+
+import { GradientIcon, GradientText } from 'app/components/core'
+import { useFeatureFlag } from 'app/hooks/useRemoteConfig'
+import type { StylesProp } from 'app/styles'
+import { makeStyles } from 'app/styles'
+
+type ScreenHeaderProps = {
+  children?: ReactChild
```

Comment:
Is `ReactChild` the preferred way of doing this? Could use ReactNode to allow multiple children right?